src/components/atoms/InputField/InputField.tsx

```diff
@@ -55,6 +55,8 @@ interface InputFieldProps
   register: UseFormRegister<AnyForm> | (() => void);
   rules?: RegisterOptions;
   name?: string;
+  onLabelClick?: () => void;
+  label?: string;
```

Comment:
Few questions that might need a understanding and a decision.

These two props have been added and removed in few of your PRs
What is the final verdict?
Should `InputField` from `src/components/atoms` use be expanded?
There are already two new ones for admin and attendee side respectively.


This file already has a comment about this:
```tsx
/**
 * @deprecated Please use the appropriate attendee or admin Input
 * @see src/components/admin/Input
 * @see src/components/attendee/Input
 */
```

Reply:
i've reverted changes in `InputField` and removed usages of this component in the new attendee files [04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)